docs-source/index.md

```diff
@@ -152,9 +151,37 @@ var allClients = await apiClient.Clients.GetAllAsync();
 
 ### Client reuse and thread safety
 
-Neither the `ManagementApiClient` or the `AuthenticationApiClient` are threadsafe, as they store the result of the last call to an API (obtainable by the `GetLastApiInfo` method). It's ok to reuse them in the same thread to do subsequent calls, but they shouldn't be created as a singleton instance used across multiple threads. 
+`ManagementApiClient` and `AuthenticationApiClient` are thread-safe with the exception of the `GetLastApiInfo` method.
 
-Both clients are really lightweight to instantiate, so creating a new instance every time they are needed shouldn't be a concern.
+While the client opjects are lightweight to instantiate by default they create their own ApiConnection object which creates it's own HttpClient.  In order to best utilize HTTP connections the HttpClient should be shared as much as possible so it can perform the necessary thread-pooling.
```

Comment:
```suggestion
While the client objects are lightweight to instantiate by default they create their own `ApiConnection` object which creates its own `HttpClient`. In order to best utilize HTTP connections, the `HttpClient` should be shared as much as possible so it can perform the necessary thread-pooling.
```